<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-ECVQGC53" failures="2" tests="5" name="Test1" time="50.385" errors="0" timestamp="2022-08-05T17:25:56 IST">
  <testcase classname="com.qa.TestLayer.LoginPageTest" name="VerifyLoginWithValidCredential" time="4.015">
    <failure type="java.lang.AssertionError" message="expected [https://qaadmin.onzway.com/dashboards] but found [https://qaadmin.onzway.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://qaadmin.onzway.com/dashboards] but found [https://qaadmin.onzway.com/]
at com.qa.TestLayer.LoginPageTest.VerifyLoginWithValidCredential(LoginPageTest.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- VerifyLoginWithValidCredential -->
  <testcase classname="com.qa.TestLayer.LoginPageTest" name="VerifyLoginWithInvalidCredential" time="3.689"/>
  <testcase classname="com.qa.TestLayer.LoginPageTest" name="VerifyEmailAcceptingInput" time="0.124">
    <failure type="java.lang.AssertionError" message="did not expect [aksjRdtt444@###nnfnn] but found [aksjRdtt444@###nnfnn]">
      <![CDATA[java.lang.AssertionError: did not expect [aksjRdtt444@###nnfnn] but found [aksjRdtt444@###nnfnn]
at com.qa.TestLayer.LoginPageTest.VerifyEmailAcceptingInput(LoginPageTest.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- VerifyEmailAcceptingInput -->
  <testcase classname="com.qa.TestLayer.LoginPageTest" name="VerifyPasswordAcceptingInput" time="0.079"/>
  <testcase classname="com.qa.TestLayer.LoginPageTest" name="VerifyForgotPasswordLink" time="0.533"/>
</testsuite> <!-- Test1 -->
